src/sessions/model/Session.ts

```diff
@@ -1,4 +1,5 @@
 export default interface Session {
   sessionId: string;
   userId: string;
+  apiAccountId: string;
```

Comment:
@ryankazokas This field should be optional since a user request for a session does not return this property.